integration-tests/azure-keyvault/src/main/java/io/quarkiverse/azure/keyvault/secret/it/KeyVaultSecretResource.java

```diff
@@ -0,0 +1,69 @@
+package io.quarkiverse.azure.keyvault.secret.it;
+
+import static jakarta.ws.rs.core.MediaType.TEXT_PLAIN;
+
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+
+import jakarta.inject.Inject;
+import jakarta.ws.rs.GET;
+import jakarta.ws.rs.Path;
+import jakarta.ws.rs.Produces;
+
+import org.jboss.logging.Logger;
+
+import com.azure.security.keyvault.secrets.SecretAsyncClient;
+import com.azure.security.keyvault.secrets.SecretClient;
+import com.azure.security.keyvault.secrets.models.SecretProperties;
+
+@Path("/keyvault")
+public class KeyVaultSecretResource {
+    private static final Logger LOG = Logger.getLogger(KeyVaultSecretResource.class);
+
+    public final static String TEXT = "Quarkus Azure Key Vault Extension is awsome";
+    private static final String SYNC_PARAM = "synkv" + System.currentTimeMillis();
+    private static final String ASYNC_PARAM = "asynckv" + System.currentTimeMillis();
+
+    @Inject
+    SecretClient secretClient;
+
+    @Inject
+    SecretAsyncClient secretAsyncClient;
+
+    @GET
+    @Path("sync")
+    @Produces(TEXT_PLAIN)
+    public String testSync() {
+        LOG.info("Testing SecretClient by creating secret: " + SYNC_PARAM);
+        //Put parameter
+        secretClient.setSecret(SYNC_PARAM, TEXT);
+        //Get parameter
+        return secretClient.getSecret(SYNC_PARAM).getValue();
+    }
+
+    @GET
+    @Path("list")
```

Comment:
Suggest to rename to `syncList` or something similar to indicate it's a sync operation. Besides, add a relative test in `KeyVaultSecretResourceTest.java`.

Reply:
See https://github.com/quarkiverse/quarkus-azure-services/pull/218/commits/af62cb73395c168d394f5e07b191eb9dd46a277a.